manifests/a/AkelPad/AkelPad/4.9.8-x64/AkelPad.AkelPad.yaml

```
@@ -1,7 +1,7 @@
 PackageIdentifier: AkelPad.AkelPad
-PackageVersion: 4.9.8
-PackageName: Akelpad Editor
-Publisher: Akelpad
+PackageVersion: 4.9.8-x64
+PackageName: AkelPad
+Publisher: AkelPad
 License: BSD 2 Clause
```

```suggestion
 License: BSD 3 Clause
```
Since the link is to the BSD 3 clause.